src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -157,11 +163,15 @@ const Notification = ({ count, disabled }) => {
                 src="${REPL_ACCOUNT}/widget/DIG.Button"
                 props={{
                   href: "/notifications",
-                  fill: "outline",
                   variant: "secondary",
                   label: "See all",
                   size: "small",
-                  style: { width: "100%" },
+                  style: {
+                    width: "100%",
+                    color: "#604CCB",
+                    borderColor: "#E3E3E0",
+                    background: "#FDFDFC",
+                  },
```

Comment:
```suggestion
                  style: {
                    width: "100%",
                  },
```

Reply:
This has been addressed in the latest commits.